src/Storefront/Resources/views/storefront/element/cms-element-product-description-reviews.html.twig

```diff
@@ -1,4 +1,4 @@
-{% block element_product_desciption_reviews %}
+{% block element_product_description_reviews %}
```

Comment:
changing a block name is a breaking change, therefore you should create a new inner block with the new name and deprecate the old block:
```twig
{# @deprecated tag:v6.5.0 - block element_product_desciption_reviews will be removed, use  block element_product_description_reviews instead #}
{% block element_product_desciption_reviews %}
    {% block element_product_description_reviews %}
    ...
    {% endblock %}
{% endblock %}
```

Reply:
@keulinho , I've added the inner blocks and deprecated the old block. Because of the extra inner blocks, the indentation of almost all blocks is changed. That's why it's more challenging to see the diff.